ProcessLib/PhaseField/CreatePhaseFieldProcess.cpp

```diff
@@ -56,24 +55,24 @@ std::unique_ptr<Process> createPhaseFieldProcess(
         auto per_process_variables = findProcessVariables(
             variables, pv_config,
             {//! \ogs_file_param_special{prj__processes__process__PHASE_FIELD__process_variables__phasefield}
-            "phasefield",
+             "phasefield",
              //! \ogs_file_param_special{prj__processes__process__PHASE_FIELD__process_variables__displacement}
```

Comment:
The parameter documentation file  is missing.  You can run ``scripts/doc/get-project-params.sh . | scripts/doc/create-docu-file-stubs.sh`` to check or  generate  parameter documentation files.